src/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_executable_network/exec_graph_info.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "behavior/ov_executable_network/exec_graph_info.hpp"
+
+#include "ie_plugin_config.hpp"
+#include <common_test_utils/test_constants.hpp>
+
+using namespace ov::test::behavior;
+namespace {
+const std::vector<ov::element::Type_t> netPrecisions = {
+        ov::element::i16,
+        ov::element::u8,
+        ov::element::f32
```

Comment:
shouldn't we check f16 precision? it should be supported for weights

Reply:
we are checking it but as an unsupported conversation. Will enable it in https://github.com/openvinotoolkit/openvino/pull/9282